pkg/lifecycle/daemon/routes_navcycle_completestep.go

```diff
@@ -37,43 +36,25 @@ func (d *NavcycleRoutes) completeStep(c *gin.Context) {
 			return
 		}
 
-		currentState, err := d.StateManager.TryLoad()
-		if err != nil {
-			c.AbortWithError(500, err)
-			return
-		}
-
 		debug.Log("event", "check.stepAlreadyComplete")
-		lifecycle := currentState.Versioned().V1.Lifecycle
-		if lifecycle == nil {
-			lifecycle = &state.Lifeycle{
-				StepsCompleted: make(map[string]interface{}),
-			}
-		}
-		_, stepAlreadyComplete := lifecycle.StepsCompleted[step.Shared().ID]
-
 		progress, ok := d.StepProgress.Load(step.Shared().ID)
-		shouldExecute := !ok || progress.Detail == `{"status":"success"}` && !stepAlreadyComplete
-
-		debug.Log("shouldExecute", shouldExecute)
+		shouldExecute := !ok || progress.Detail == `{"status":"success"}`
 		if shouldExecute {
```

Comment:
What do you do if this is false?
Do we want to, you know, not execute the step?

Reply:
no we're always going to "re-execute"

Reply:
Because it looks like we execute either way

Reply:
that's the intent. i'm removing to avoid confusion